exercises/simple-cipher/src/test/java/SimpleCipherStepTwoTest.java

```diff
@@ -1,79 +1,74 @@
 import org.junit.Before;
-import org.junit.Test;
 import org.junit.Ignore;
+import org.junit.Test;
 
 import static org.junit.Assert.assertEquals;
 
-public class SubstitutionCipherTest {
-
+/**
+ * Step 2: Specify key and use that for shift distance : substitution cipher
+ */
+public class SimpleCipherStepTwoTest {
+    private Cipher cipherWithSetKey;
     private static final String KEY = "abcdefghij";
```

Comment:
Another opportunistic improvement suggestion (feel free to ignore, if you disagree/don't want to)...

Personally, I'm not a fan of using constants as fixtures in tests.  The fact that it is a value used in a test case is far far more important than the incidental fact that it's constant (all String literals *are* constant in Java).

Seems to me that `"abcdefghij"` is playing the role of an important piece of data for *each* test case (exactly like `cipherWithSetKey`) ... call it `key`?